platform/osx/display_server_osx.mm

```diff
@@ -2317,12 +2317,16 @@ static void displays_arrangement_changed(CGDirectDisplayID display_id, CGDisplay
 			window_set_flag(WindowFlags(i), true, id);
 		}
 	}
+
+	return id;
+}
+
+void DisplayServerOSX::show_window(const WindowData &wd, uint32_t p_flags) {
```

Comment:
It's `... show_window(WindowID p_id ...` on other platforms, probably should be the same here. And definition is missing from `display_server_osx.h`.

```obj-c++
void DisplayServerOSX::show_window(WindowID p_id, uint32_t p_flags) {
	WindowData &wd = windows[p_id];
	....
```

Reply:
oh yeah sorry, i messed up, i was distracted and wrote without thinking too much. I should be more careful while writing code i can't compile eheh. I'll fix it immediately!